packages/opentelemetry-api/src/api/propagation.ts

```diff
@@ -23,12 +23,16 @@ import { ContextAPI } from './context';
 
 const contextApi = ContextAPI.getInstance();
 
+const GLOBAL_PROPAGATION_API_KEY = Symbol.for(
```

Comment:
I would prefer to keep these symbols/keys in one place, either `symbols.ts` or `global_keys.ts`. WDYT?

Reply:
I made a global utils file. PTAL